app/windowsInit.js

```diff
@@ -9,7 +9,25 @@ const spawn = childProcess.spawn
 const spawnSync = childProcess.spawnSync
 const execSync = childProcess.execSync
 const app = electron.app
-const appUserModelId = 'com.squirrel.brave.Brave'
+const Channel = require('./channel')
+let appUserModelId = 'com.squirrel.brave.Brave'
+switch (Channel.channel()) {
+  case 'nightly':
```

Comment:
kind of a lame nit-pick; you could store the text representation of the channels in an enum-like format in channel.js:
```js
const channelStrings = {
  RELEASE: 'dev',
  BETA: 'beta',
  // ..
}
and then export / import from this file.  Will make updating the constants easier :smile: